bigquery/bq_schema.json

```diff
@@ -0,0 +1,193 @@
+{ "BigQuery Schema":
+[
+  {
+    "mode": "REQUIRED",
+    "name": "last_updated_ts",
+    "type": "TIMESTAMP"
+  },
+  {
+    "mode": "REQUIRED",
+    "name": "last_updated",
+    "type": "STRING"
+  },
+  {
+    "mode": "NULLABLE",
+    "name": "E0",
```

Comment:
Let's finally start using sensible names for these fields now that we are doing greenfield work. Yea, I know annoying, you'll have to add a mapping.

Reply:
Lets open a separate story for this, the current data format is documented and agreed and has compatibility with all parts of the system, if we wish to rename the keys we should handle separately and refactor all instances.